base/cpuid.jl

```diff
@@ -88,14 +89,29 @@ function normalize_arch(arch::String)
     return arch
 end
 
+const ALL_FEATURES = let
+    get_features(prefix::String) =
+        getfield.(Ref(@__MODULE__), filter(n -> startswith(String(n), prefix), (names(@__MODULE__; all=true))))
```

Comment:
If you're wondering why I had to move `include("binaryplatforms.jl")` around in `Base.jl`, it looks like something here (`filter`?) needs a `sort!` method.

If you know better ways to collect the `JL_*` symbols, please share!  I tried putting them in (yet) another submodule at least to avoid `names(@__MODULE__)`, but wasn't sure that was a better solution anyway.

Reply:
Ok, I simplified the code to avoid the funny `get_features` function.  Now I also collect the features only for the current platform instead of all of them, no need for that.  However we still depend on the `Base.Sort` module (because of `unique!`?)